src/Codec/Xlsx/Types/Drawing.hs

```diff
@@ -217,6 +217,21 @@ data Anchor p g = Anchor
     } deriving (Eq, Show, Generic)
 instance (NFData p, NFData g) => NFData (Anchor p g)
 
+data GenericDrawing p g = Drawing
```

Comment:
and why do we need this code block move?

Reply:
Instances defined after TH splices are not seen before them: https://gitlab.haskell.org/ghc/ghc/-/wikis/migration/9.0#the-order-of-th-splices-is-more-important

In this case I had to move `makeLenses` after `Default` instance, but then this type started giving errors as well, so I had to move it before splice as well.